db/schema.rb

```diff
@@ -1,18 +1,22 @@
-# This file is auto-generated from the current state of the database. Instead
```

Comment:
Like the comment says this is *auto-generated* so there's no point in removing these comments since they'll likely be back later on. You should never *manually* edit the schema file of your rails application. Running the migrations will take care of that